tests/components/kem/conftest.py

```diff
@@ -0,0 +1,85 @@
+"""Module for testing the KEM integration in Home Assistant."""
+
+from typing import Any
+from unittest.mock import patch
+
+from aiokem import AioKem
+import pytest
+
+from homeassistant.components.kem.const import CONF_REFRESH_TOKEN, DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from tests.common import MockConfigEntry, load_json_value_fixture
+
+
+@pytest.fixture(name="homes")
+def kem_homes_fixture() -> list[dict[str, Any]]:
+    """Create sonos favorites fixture."""
+    return load_json_value_fixture("homes.json", "kem")
```

Comment:
technically this does I/O so fixture should become async and wrap the call in hass.async_add_executor_job

Reply:
So far I have not seen this happen in tests, is this something we want todo?

Reply:
Should we fix this in tests/common.py so everyone gets it?

Reply:
If there indeed is benefit of doing this I am happy to help looking into this, but then we should fix this globally imo